index.html

```diff
@@ -4,6 +4,8 @@
   <title>Weclome to XYZ</title>
   </head>
   <body>
+
+  <h1>Hey Customer! Welcome to our shop</h1>
```

Comment:
This is good, but could you add the message "We are ready to serve you" and make the color red